docs/samples/advanced_config/aggregation_job_driver.yaml

```diff
@@ -87,6 +87,17 @@ worker_lease_clock_skew_allowance_secs: 60
 # (required)
 maximum_attempts_before_failure: 10
 
+# Timeout to apply when establishing connections to the helper for HTTP requests, in seconds. See
+# https://docs.rs/reqwest/latest/reqwest/struct.ClientBuilder.html#method.connect_timeout for
+# details. (optional)
+http_request_connection_timeout_secs: 10
```

Comment:
This may be getting ahead of ourselves, but what do you think about naming these settings based on the type of request they apply to? We can kinda-sorta get there since each job type has one type of request[1], but that requires operators to know the type of request each job sends.

[1] Sort of -- I suppose we might eventually view "aggregate initialization" as separate from "aggregate continuation", giving the agg job driver two kinds of requests. This practically doesn't matter right now, since we aren't deplying multi-step VDAFs at the moment.

Reply:
I think it's better this way. The nice thing about this layout is that we can add the new options to `struct JobDriverConfig` and get them for free in both the collection and aggregation job drivers. Also, these values are currently used to create a `reqwest::Client` that is then used for all HTTP requests made by the job driver. So saying the timeouts apply to all HTTP requests is actually accurate.